src/main.js

```diff
@@ -1,35 +1,39 @@
 const bigRecipeCard = document.querySelector('.recipe-pop-up');
 const allRecipesDisplay = document.querySelector('.all-recipes-display');
+const userPageDisplay = document.querySelector('.user-window');
+const favoriteRecipesDisplay = document.querySelector('.favorite-recipes');
 const nav = document.querySelector('nav');
 //data instantiation
 const currentUser = new User(generateRandomUser());
+// const currentUser = new User(usersData[0]);
 const instantiatedRecipes = recipeData.map(recipe => new Recipe(recipe));
 //onload 
 window.onload = handleLoad();
 //event listening
 allRecipesDisplay.addEventListener('click', smallRecipeHandler);
+favoriteRecipesDisplay.addEventListener('click', smallRecipeHandler);
 bigRecipeCard.addEventListener('click', bigRecipeHandler);
 nav.addEventListener('click', navHandler);
```

Comment:
We can definitely connect at some point about making one large event listener - I think body.addEventListener might have some merit here, like you said. I say the less we have floating around, the better.